cob_bringup/robots/cob4-1.launch

```diff
@@ -2,12 +2,12 @@
 <launch>
 
 	<include file="$(find cob_bringup)/robots/cob4-1.xml" >
-		<arg name="cob4-1-b1" value="cob4-2-b1"/>
-		<arg name="cob4-1-t1" value="cob4-2-t1"/>
-		<arg name="cob4-1-t2" value="cob4-2-t2"/>
-		<arg name="cob4-1-t3" value="cob4-2-t3"/>
-		<arg name="cob4-1-s1" value="cob4-2-s1"/>
-		<arg name="cob4-1-h1" value="cob4-2-h1"/>
+		<arg name="cob4-1-b1" value="cob4-1-b1"/>
```

Comment:
I think that it is wrong, this value is also used to define the address(hostnames) of the machines https://github.com/ipa320/cob_robots/blob/indigo_dev/cob_bringup/robots/cob4-1.xml#L22